### DISCUSSION TITLE
How to change the colormap of the pseudosection?

### DISCUSSION THREAD

**Discussion by jcmefra** [gimli-org/pyGIMLi#705] [Q&A]
Hello! I'm curious if it's possible to change the colormap of the pseudosection, I'm doing:

`ert.show(data, cmap='jet', style="a-sepm")`

And I'm still getting the spectral colormap:

![image](https://github.com/gimli-org/gimli/assets/64992303/8af7ad7d-e92a-42e0-b618-cb380bb379ea)

Also the scheme is multiple gradient, but the pseudosection shows "SL", is it possible to modify it? Thank you in advance.

**Comment by halbmy** [comment]
For the colormap, you have to use `cMap` with a big M. By the way, there was an error in the `a-sepm` that I just fixed.

SL is a Schlumberger type array, of which the arrays in the multiple-gradient scheme are a subset, or maybe the other way round: a (symmetric) Schlumberger array is included in the multi-gradient sequency. We could name it GR but we can hardly make a difference. After all, it is just a name and you can change it using `ax.set_xticklabel`.

**Reply by jcmefra** [reply]
Thanks for the information. I just updated pygimli and I'm getting this error:

`ImportError: cannot import name 'RVector3' from 'pygimli.core.core'` when doing ` import pygimli as pg`, so I couldn't try the modifications you suggested. How to solve this? Should I just reinstall? Thank you.

Edit: I just reinstalled and it worked, but ax.set_xticklabel is not working. It's still showing SL:

![image](https://github.com/gimli-org/gimli/assets/64992303/6df524bc-d9e1-4c9c-93d2-48ada46d21d5)

**Reply by halbmy** [reply]
Of course, I meant `ax.set_yticklabels()`, maybe first getting it with `ax.get_yticklabels()` and replacing SL.

**Comment by halbmy** [comment]
Can you please check which pyGIMLi and pgcore versions you have? I recommend updating to v1.5.0 and try again.

To make use of the `a-sepm` fix, you will have to use the newest git version https://www.pygimli.org/installation.html#staying-up-to-date
